### PR TITLE
Fix/updated vda login

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Updated profile event listener
- FIxes minor issue with avatar display

### CODE REVIEW

File: components/account/src/lib-components/vda-account.vue [hunk 1]
```diff
@@ -135,10 +139,12 @@ export default defineComponent({
           contextName: this.contextName,
           logo: this.logo,
         });
-        this.profile = this.$VeridaHelper.profile;
-        this.profile.avatar = this.$VeridaHelper.profile.avatar.uri;
-        this.profile.did = this.$VeridaHelper.did;
-        this.onSuccess(this.$VeridaHelper.context);
+        // this.profile = this.$VeridaHelper.profile;
```
Comment (nick-verida) [inline components/account/src/lib-components/vda-account.vue:142]: remove commented out code

File: lerna-debug.log [hunk 2]
```diff
@@ -0,0 +1,7 @@
+36 error Error: Command failed with exit code 128: git tag @verida/vue-account@0.1.3 -m @verida/vue-account@0.1.3
```
Comment (nick-verida) [inline lerna-debug.log:1]: Remove this file

File: package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
 	],
 	"scripts": {
 		"install": "lerna exec -- yarn install",
-		"test:unit": "vue-cli-service test:unit",
+		"test:unit": "vue-cli-service test:unit --env=./tests/custom-test-env.js",
```
Comment (nick-verida) [inline package.json:14]: This file doesn't seem to be in the commit

**Review by nick-verida — CHANGES_REQUESTED** [review]
A few minor changes

**Comment by cmcWebCode40** [comment]
@nick this PR has been updated with :

- [ ]  Requested changes
- [ ] Refactor to remove vuex store
- [ ] separation of concerns in vda components
- [ ] test setup

FYI: the test scripts fail ATM the moment issue can be found here https://github.com/verida/verida-vue-components/issues/29#issuecomment-1039524603

but this PR can be merged because it's currently a blocker for https://github.com/verida/demo-credential-issuer/pull/38

**Review by nick-verida — APPROVED** [review]
Made a change to fix the TextEncoder issue

**Pull request merged**